cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -237,3 +237,79 @@ def test_service_cloud_tenant_quota_with_default_entry_point(request, appliance,
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(provision_request.remove_request)
     assert provision_request.row.reason.text == "Quota Exceeded"
+
+
+@pytest.fixture(scope="function")
+def testing_instance(appliance, provider, small_template, setup_provider):
```

Comment:
_Optional_ since tests always start with `test` I think it is better to just call this `instance`, what do you think?